src/pages/talks/2018-03-the-client-side-revolution.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import Talk from './_talk';
+
+const TalkImplementation = () => (
+  <Talk>
+    {{
+      id: 'the-client-side-revolution',
+      subtitle: (
+        <div>
+          <a href="https://www.twitter.com/kylemathews" target="_blank">
+            Kyle Mathews
+          </a>{' '}
+          from{' '}
+          <a href="https://www.gatsbyjs.org/" target="_blank">
+            Gatsby
+          </a>
+        </div>
+      ),
+      videoUrl: 'https://www.youtube.com/embed/Bq31S6h6TwE',
+      content: (
+        <div>
+          <p>
+            Building websites has changed dramatically over the last few years.
+            In this session we’ll go on a history trip to discover where we came
+            from, why React is a such a big deal, and where we might be going
+            soon. We’ll discuss what a website is and why we use the tools we do
+            to build them and how this is the best of all times to be building
+            for the web.
+          </p>
+          <p>Presented at Reactathon 2018 in San Francisco.</p>
+          <p style={{ fontSize: 14 }}>
```

Comment:
Maybe using "<small>" would be better, if not I think it's ok.

Reply:
For styles? I think this is pretty standard for JSX, but then again I'm not familiar with your CSS in JS format. 

It's a single inline element so I think we'll be fine.

Reply:
Sorry, typed incorrectly. I mean, using small tags instead of p with font-size 14px.

Reply:
I think the `small` tag is too small. I'll just stick with 14px.